docs/src/spec/expressions/name.md

```diff
@@ -9,11 +9,12 @@ A *name expression* resolves to a local variable.
 Example:
 
 ```python
```

Comment:
Not needed for this PR but we should probably switch from `python` syntax highlighting to `rust` syntax highlighting or else the snippets in the docs will look less good. I think that will also require a tweak in some script that performs the tests on top of the docs snippets.

Reply:
mdbook strips out lines that start with `#` in rust-formatted blocks, so I added a simple custom fe syntax highlighter instead.

Reply:
Sweet